### ISSUE TITLE
Does not work with salt 2017.7 (nitrogen)

### ISSUE THREAD

**Issue by philpep** [saltstack-formulas/epel-formula#37]
Hello, the formula doesn't seems to work with salt 2017.7 because osmajorrelease grain is now an integer so this code: ``if grains['osmajorrelease'][0] == '5'`` trigger a ``Rendering SLS 'base:epel' failed: Jinja variable int object has no element 0``.

**Comment by fujexo** [comment]
On 2017.07, the following works but I'm not sure if it still works on older versions and I currently do not have the time to test it:

```
- if grains['osmajorrelease'][0] == '5'
+ if grains['osmajorrelease'] == 5
```

**Comment by dseira** [comment]
@fujexo With that change, minions with older versions won't work. There is a new commit that should work with all versions.

**Comment by philpep** [comment]
Closed by #38
